docs/modules/engines/pages/introduction.adoc

```diff
@@ -0,0 +1,21 @@
+
+= Correlation Engines
+
+Correlation engines are the underlying processes that drive ALEC.
+Two engines are implemented by default: xref:clustering.adoc[Clustering] and xref:deeplearning.adoc[Deep Learning].
+Both extend the cluster engine, which builds network topology graphs based on information provided by your datasource (see xref:datasources:overview.adoc[ALEC Datasources]).
```

Comment:
```suggestion
Both extend the correlation engine, which builds network topology graphs based on information provided by your datasource (see xref:datasources:overview.adoc[ALEC Datasources]).
```

Reply:
We may want to talk to the team about the naming here, since the underlying cluster engine is different from the cluster*ing* engine. I'll note it for tomorrow.